cms/tests/test_plugins.py

```diff
@@ -758,8 +758,8 @@ def test_inheritplugin_media(self):
         """
 
         inheritfrompage = api.create_page('page to inherit from',
```

Comment:
As part of flake8 fixes, do you think there would be any consensus to use hanging indent instead?
```
inheritfrompage = api.create_page(
    'page to inherit from',
    'nav_playground.html',
    'en'
)
```
I always found this style an efficient use of whitespace since it often forces large chunks to the right side.